extensions/mock-credentials-verifier/build.gradle.kts

```diff
@@ -7,12 +7,15 @@ val edcGroup: String by project
 val jupiterVersion: String by project
 val mockitoVersion: String by project
 val assertj: String by project
+val identityHubGroup: String by project
+val identityHubVersion: String by project
 
 dependencies {
     implementation("${edcGroup}:identity-did-spi:${edcVersion}")
```

Comment:
Could you change this one to `indentityHubVersion` too?

Reply:
identity-did-spi comes from EDC